nservicebus/sqlserver/operations-scripting.md

```diff
@@ -31,6 +31,10 @@ snippet:sqlserver-powershell-nativesend
 
 snippet:sqlserver-nativesend-usage
 
+The `MessageTypeToSend` dictionary value represents the message .NET type, by default the endpoint receiving the message will use that message type to deserialize the incoming message. Depending on the [versioning](/samples/versioning/sample.md) strategy the type can be expressed in 2 different ways:
```

Comment:
I agree with @SimonCropp. 
@mauroservienti - I think we can try and add that info into this doco: http://docs.particular.net/nservicebus/messaging/message-owner

Thoughts?